src/Kodify/BlogBundle/Resources/views/Post/List/index.html.twig

```diff
@@ -3,10 +3,10 @@
 {% block content %}
 
     {% for post in posts %}
-        <div class="panel panel-default">
+        <div class="panel panel-default col-md-6">
```

Comment:
Well, I hope that this bootstrap class will mantain the proportions of the div's and make a responsive reaction ;)